### PR TITLE
Revised README to be more formal

### PR DESCRIPTION
**Pull request by sidagrawal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ VectorTiles
 
 What are VectorTiles?
 ---------------------
-Invented by [Mapbox](https://www.mapbox.com/), they are a combination of the
-ideas of finite-sized tiles and vector geometries. Mapbox maintains the
+Invented by [Mapbox](https://www.mapbox.com/), `vectortiles` are a combination
```
Comment (fosskers) [inline README.md]:
```suggestion
Invented by [Mapbox](https://www.mapbox.com/), VectorTiles are a combination
```

File: README.md [hunk 2]
```diff
@@ -206,9 +207,9 @@ issue](https://github.com/mapbox/vector-tile-spec/issues/80) reports another
 
 #### Sanity Checks / Error Handling
 
-The protobuf data can be malformed in a number of ways. How much sanity
-checking you wish to do while decoding depends on how much performance you
-can sacrifice. For instance, here is a constraint found in the spec
+The protobuf data can be malformed in a number of ways. The amount of sanity
+checks while decoding depends on willingness to sacrifice performance.
```
Comment (fosskers) [inline README.md]:
```suggestion
checking that occurs while decoding depends on willingness to sacrifice performance.
```

**Review by fosskers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sidagrawal) [reply README.md]: Made the change

File: README.md (hunk 2 above)
Comment (sidagrawal) [reply README.md]: Made the change

**Review by fosskers — APPROVED** [review]

**Pull request merged**